### ISSUE TITLE
Command plugins provokes a crash

### ISSUE THREAD

**Issue by mohierf** [mickem/nscp#367]
With a simple configuration, running `nscp test` the *plugins* command provokes a crash.

```
C:\Program Files\NSClient++>nscp test
L     client Module: CommandClient
L     client Command:
L     client Extra Query:
L     client Mode: 3
L     client Boot: 1
L     client Load All: 0
L     client Warning module and boot specified only THAT module will be loaded
L     client Arguments:
D       core NSClient++ 0.5.0.62 2016-09-14 x64 Loading settings and logger...
D       core Settings not ready so we cant lookup: base-path
D       core Settings not ready so we cant lookup: exe-path
D   settings Boot.ini found in: C:\Program Files\NSClient++/boot.ini
D       core Settings not ready so we cant lookup: exe-path
D   settings Activating: ini://${exe-path}/nscp.ini
D   settings Creating instance for: ini://${exe-path}/nscp.ini
D       core Settings not ready so we cant lookup: exe-path
D   settings Loading: C:\Program Files\NSClient++/nscp.ini
D       core NSClient++ 0.5.0.62 2016-09-14 x64 booting...
D       core Booted settings subsystem...
D       core On crash: restart: NSCP
D       core Archiving crash dumps in: C:\Program Files\NSClient++/crash-dumps
D       core booting::loading plugins
D       core Found: CheckSystem
D       core Found: NSCAClient
D       core Found: PythonScript
D       core Found: Scheduler
D       core Found: WebServer
D       core C:\Program Files\NSClient++/modules\CheckSystem.dll.dll
D       core adding C:\Program Files\NSClient++/modules\CheckSystem.dll
D       core C:\Program Files\NSClient++/modules\NSCAClient.dll.dll
D       core adding C:\Program Files\NSClient++/modules\NSCAClient.dll
D       core C:\Program Files\NSClient++/modules\PythonScript.dll.dll
D       core adding C:\Program Files\NSClient++/modules\PythonScript.dll
D       core C:\Program Files\NSClient++/modules\Scheduler.dll.dll
D       core adding C:\Program Files\NSClient++/modules\Scheduler.dll
D       core C:\Program Files\NSClient++/modules\WebServer.dll.dll
D       core adding C:\Program Files\NSClient++/modules\WebServer.dll
D       core Loading plugin: CheckSystem
D       core Loading plugin: NSCAClient
D       core Loading plugin: PythonScript
D     python Looking for: sample
D     python Looking for: sample.py
D     python Looking for: C:\Program Files\NSClient++\scripts\python\sample
D     python Looking for: C:\Program Files\NSClient++\scripts\python\sample.py
D     python Adding script: my_py (C:\Program Files\NSClient++\scripts\python\sample.py)
D     python boot python
D     python Prepare python
D     python init python
D     python Loading python script: C:\Program Files\NSClient++\scripts\python\sample.py
D     python Lib path: C:\Program Files\NSClient++\scripts\python\lib
L     python My god: its full of stars: true
L     python Adding a simple function/cmd line
D       core Loading plugin: Scheduler
D       core Loading plugin: WEBServer
D        web Allowed hosts definition: 127.0.0.1(255.255.255.255)
D        web Using certificate: C:\Program Files\NSClient++/security/certificate.pem
D        web Loading webserver on port: 8443s
D  w32system Loading counter: disk_queue_length_0 C: D: E: = \\FVC320\Disque physique(0 C: D: E:)\Pourcentage
isque
D  w32system Loading counter: disk_queue_length__Total = \\FVC320\Disque physique(_Total)\Pourcentage du temp
D       core NSClient++ - 0.5.0.62 2016-09-14 Started!
D       core C:\Program Files\NSClient++/modules\CommandClient.dll.dll
D       core adding C:\Program Files\NSClient++/modules\CommandClient.dll
D       core Loading plugin: CommandClient...
D        cli Enter command to execute, help for help or exit to exit...
plugins
D       core Loading C:/Program Files\NSClient++\modules\CheckDisk.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckEventLog.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckExternalScripts.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckHelpers.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckLogFile.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckNet.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckNSCP.dll
D       core Found cached CheckSystem
D       core Loading C:/Program Files\NSClient++\modules\CheckTaskSched.dll
D       core Loading C:/Program Files\NSClient++\modules\CheckWMI.dll
D       core Found cached CommandClient
D       core Loading C:/Program Files\NSClient++\modules\DotnetPlugins.dll
INF: Detected crash...
INF: Running: "C:\Program Files\NSClient++\reporter.exe" "archive" "C:\Users\mohierf\AppData\Local\Temp\\4f63
ab6-b9de-0d458e4b70f6.dmp" "NSClient++" "0.5.0.62" "2016-09-14" "C:\Program Files\NSClient++/crash-dumps"
ERR: Service not started, not restarting...
```

**Comment by mickem** [comment]
Yeah, this is one of those things... it is related to DotNet module which apparently for some reason crashes at some people, I have yet to figure out why...

**Comment by mickem** [comment]
(So a workaround is to delete the DotnetPlugins.dll module, until a fix is available...

**Comment by mohierf** [comment]
Ok

**Comment by mickem** [comment]
This should be fixed in 0.5.1
